src/backend/joanie/core/serializers/client.py

```diff
@@ -736,6 +736,46 @@ class Meta:
         read_only_fields = fields
 
 
+class NestedCourseOrderSerializer(serializers.ModelSerializer):
+    """Order model serializer for the Course model"""
+
+    id = serializers.CharField(read_only=True)
```

Comment:
We could to a little peer review to think about which fields to include in this serializer